l10n/en/firefox/whatsnew/whatsnew-fx81.ftl

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/81.0/whatsnew/all/
+
+# html page title
+whatsnew81-page-title = What’s new with { -brand-name-firefox }
+
+# Main title
+whatsnew81-main-title = A new { -brand-name-firefox } for every device
+whatsnew81-main-intro = Whatever operating system you’ve got, there’s a better-than-ever, non-profit-backed mobile browser to put on it.
+whatsnew81-send-form-label = Enter your email for a download link
+whatsnew81-send-form-label-alt = Enter your email or phone number for a download link
+whatsnew81-scan-this-qr-code = Scan this QR code to download
+
+# content blocks below hero
+whatsnew81-the-new-firefox-for-android = The NEW { -brand-name-firefox } for { -brand-name-android } is here
+
+# "Weirdo" refers to features that are unusual in a delightful way. "Indie" is short slang for independent.
+whatsnew81-our-latest-for-android-has = Our latest for { -brand-name-android } has lots of fun, smart and even weirdo features – like a moveable search bar. Because we’re an indie browser. We care about your thumbs.
+
+# "Phoning it in" is when someone doesn't fully commit to doing something and just does the minimum.
+whatsnew81-no-more-phoning-it-in = No more phoning it in with { -brand-name-safari }
+whatsnew81-attention-iphone-people = Attention { -brand-name-iphone } people: you can finally set { -brand-name-firefox } as your forever go-to, get-it-done, default browser. Make sure you’ve got the latest { -brand-name-firefox } for { -brand-name-ios }.
```

Comment:
This feels a little misleading perhaps - from what I understand, you need the latest iOS update to set the default browser (which isn't quite out yet), and not the latest Firefox for iOS update?